index.bs

```diff
@@ -85,8 +88,8 @@ The DeviceOrientationSensor Interface {#deviceorientationsensor-interface}
 <pre class="idl">
   [Constructor(optional SensorOptions sensorOptions)]
   interface DeviceOrientationSensor : Sensor {
-    void populateQuaternion(Float32Array array);
-    void populateMatrix(Float32Array array);
+    void populateQuaternion(Float32Array buffer);
```

Comment:
the proposed API only populates the latest reading (same as in other sensors) it is not meant to collect series of previous readings.